edit-post/components/layout/index.js

```diff
@@ -82,6 +84,11 @@ function Layout( {
 			) }
 			{ editorSidebarOpened && <Sidebar /> }
 			{ pluginSidebarOpened && <PluginsPanel /> }
+			{
+				isMobileViewport &&
+				( publishSidebarOpened || editorSidebarOpened || pluginSidebarOpened ) &&
```

Comment:
Nit: can we extract inline variable for those 3 flags? We use it twice.

Reply:
Thanks for catching that. I pushed a fix.